codegen/templates/keyword/card_properties.j2

```diff
@@ -1,18 +1,18 @@
 {% if card.duplicate %}
     @property
     def {{card.duplicate.name}}(self):
-        '''Gets the table of {{card.duplicate.name}}'''
+        """Gets the table of {{card.duplicate.name}}."""
```

Comment:
I think the guidelines from the doc team now is to use the imperative tense, `Get` instead of `Gets`.